tests/test_coordinator.py

```diff
@@ -34,39 +35,39 @@ async def test_coordinator_async_update_with_data(mocker, coordinator):
     # Test
     await coordinator.async_refresh()
     assert coordinator.data == {
-        "alerts": {
-            "alarm_led": 0,
-            "anomalies_led": 1,
-            "device_failure": 0,
-            "device_low_battery": 0,
-            "device_no_power": 0,
-            "device_no_supervision": 0,
-            "device_system_block": 0,
-            "device_tamper": 0,
-            "gsm_anomaly": 0,
-            "gsm_low_balance": 0,
-            "has_anomaly": False,
-            "input_alarm": 0,
-            "input_bypass": 0,
-            "input_failure": 0,
-            "input_low_battery": 0,
-            "input_no_supervision": 0,
-            "inputs_led": 2,
-            "module_registration": 0,
-            "panel_low_battery": 0,
-            "panel_no_power": 0,
-            "panel_tamper": 0,
-            "pstn_anomaly": 0,
-            "rf_interference": 0,
-            "system_test": 0,
-            "tamper_led": 0,
+        q.ALERTS: {
```

Comment:
Use the actual ID and not `q.ALERTS` (applies everywhere in the code). The rationale is that you should write exactly what you expect and not use variables, otherwise if we change the value of `q.INPUTS` to something else, tests will pass but the integration will be broken. So, always use the actual `int`.